.circleci/testnet-deploy-new-chain-for-pr.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+/*
+This script updates a local Boyar config.json file to provision a new chain
+for E2E purposes on an isolated blockchain network
+
+Usage: 
+
+$ testnet-deploy-new-chain-for-pr.js <GitHub PR Link>
+
+Examples:
+
+$ testnet-deploy-new-chain-for-pr.js https://github.com/orbs-network/orbs-network-go/pull/1184
```

Comment:
Please use `$CIRCLE_PR_NUMBER` instead.

Reply:
From my checks it's not available anymore. When I started developing I printed the `env` and looked at whatever is available to work with. the only thing was `$CI_PULL_REQUESTS` - no other variable was available in the environment to work with. You can check for yourself.

Reply:
You are correct.